src/test/java/com/redhat/red/build/finder/TestUtils.java

```diff
@@ -35,10 +35,12 @@ public static File resolveFileResource(final String resourceBase, final String r
     }
 
     public static File loadFile(String file) throws IOException {
-        URL urlFile = DistributionAnalyzerTest.class.getClassLoader().getResource(file);
+        URL urlFile = Thread.currentThread().getContextClassLoader().getResource(file);
+
```

Comment:
I am not sure this change is needed ?

Reply:
The apparent bug is that it wasn't referring to itself, so I believe that should be `TestUtils.class` and not `DistributionAnalyzer.class` at the very least.

It possibly could be

```
getClass().getClassLoader().getResource(...)
```

I don't know which is the most correct, but the one there is the one that the kojiji tests are using.

Reply:
True, it should be TestUtils. That sounds like a typo. ```getClass().getClassLoader()``` sounds good to me!